Godeps/Godeps.json

```diff
@@ -1171,127 +1171,127 @@
 		},
 		{
 			"ImportPath": "github.com/gogo/protobuf/gogoproto",
-			"Comment": "v0.4-3-gc0656edd",
+			"Comment": "v0.4-3-gc0656ed",
```

Comment:
I'm no expert here, but I'm seeing quite some packages in the file that don't seem to be from standard releases (e.g docker/engine-api, coreos/pkg). Apparently even the go-restful pkg.
IMO allowing only releases could become a serious limitation wrt feature velocity (i.e. wait for an upstream release before using it in your feature).

cc @thockin - the godep god :)